packages/browser/src/node/providers/playwright.ts

```diff
@@ -185,6 +185,12 @@ export class PlaywrightBrowserProvider implements BrowserProvider {
       })
     }
 
+    // unhandled page crashes will hang vitest process
```

Comment:
this was making the tests hang https://github.com/vitest-dev/vitest/issues/7249#issuecomment-2592373055